static/index.html

```diff
@@ -48,5 +48,13 @@
 
         <script src="static/img_box.js"></script>
         <script type="module" src="static/main.js"></script>
+        <div style="margin-top: 5%;">
+        <div id="bottom_container">
+            <a id="source_link" title="Source code on GitHub" href="https://github.com/pavtiger/Parse-tables-from-PDF.git">
```

Comment:
Link to repo should not end with ".git". That is for cloning using git not for view in in browsers (however it still would work, but it's worse)